docs/config/customize.md

```diff
@@ -99,4 +99,16 @@ There are three settings you can play with to control that:
     # Which field to use in the URL, may also be for example "pk" to use the
     # primary key and not expose the username (which may be private or may change too
     # often for URL persistance)
-    USER_URL_FIELD = "username"
\ No newline at end of file
+    USER_URL_FIELD = "username"
+
+
+## Custom header and/or footer scripts
+
+You can populate the content of you own `umap/header.html` and `umap/footer.html`
```

Comment:
I'm not sure about the naming: `header.html` seems to me to be content oriented, but given it will be included in the `<head>` section, I'm tempted to call it `head.html`, `extra_head.html`…

Reply:
I choose the `header` name to make it the explicit counterpart of the existing `footer`, I don't know if editors/installers will have the knowledge to make the distinction between header/head/extra_head. I'm willing to change it if you think so.

Reply:
OK, let's go like that and see how people use it, if they use it ;)